astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -5,7 +5,8 @@
 
 from ... import units as u
 from ...tests.helper import pytest
-from ...utils.compat import NUMPY_LT_1_7, NUMPY_LT_1_8, NUMPY_LT_1_9_1
+from ...utils.compat import NUMPY_LT_1_7
+from ...utils.compat import NUMPY_LT_1_8, NUMPY_LT_1_9_1  # pylint: disable=W0611
```

Comment:
Has pytest improved enough that one can use `mark.xfail` without the quotation marks?